content/spin/kv-store.md

```diff
@@ -65,7 +65,7 @@ As previously documented, you can go ahead and [create a new Spin application fr
 <!-- @selectiveCpy -->
 
 ```bash
-$ spin new http-rust spin-key-value
+$ spin new http-rust spin-rust-key-value
```

Comment:
Given that all these blocks are under per-language tabs, I'm not sure if there's any value in having different names.  Would it be bad if we called them all plain `spin-key-value`?  I don't have strong views, more raising the question for @tpmccallum.

Reply:
Consistency is welcome across all tabs; greatly appreciated and about to be merged ...